### PR TITLE
SirChilledMrHobo's Update items.csv

### PR DESCRIPTION
**Pull request by SirChilledMrHobo**
Creator: SirChilledMrHobo
Date: 5/11/17
Version: 1 

Major Changes:
Added boat types
Added on to door types
Added on to random items

Items Added:
wartbloock 214
salmoncook 350:1
spdoor 427
bdoor 428
jdoor 429
adoor 430
ddoor 431
dodoor 431
cstick 398
carrotstick 398
sboat 444
spruceboat 444
bboat 445
birchboat 444
jboat 446
jungleboat 446
acaciaboat 447
aboat 447
darkoakboat 448
doboat 448
darkboat 448
oakboat 333
oboat 333
rfoot 414
astand 416
rawrabbit 411

### CODE REVIEW

File: items.csv [hunk 1]
```diff
@@ -7121,12 +7125,16 @@ prismarinecrystals,410,0
 priscrystals,410,0
 rabbit,411,0
 rabbitmeat,411,0
+rawrabbit,411,0
 cookedrabbit,412,0
 rabbitstew,413,0
 hasenpfeffer,413,0
 rabbitfoot,414,0
+rfoot,414,0
+foot,414,0
```
Comment (andrewkm) [inline items.csv]: I think foot is too generic, please edit and remove.

Comment (andrewkm) [inline items.csv]: rfoot on the other hand is fine.

**Comment by SirChilledMrHobo** [comment]
Removed Foot.

**Comment by andrewkm** [comment]
Please read all my comments and fix what was requested.

File: items.csv [hunk 2]
```diff
@@ -7121,12 +7125,15 @@ prismarinecrystals,410,0
 priscrystals,410,0
 rabbit,411,0
 rabbitmeat,411,0
+rawrabbit,411,0
 cookedrabbit,412,0
 rabbitstew,413,0
 hasenpfeffer,413,0
 rabbitfoot,414,0
```
Comment (SirChilledMrHobo) [inline items.csv]: Foot was removed.

**Comment by SirChilledMrHobo** [comment]
The only comments I have from you is, via email, "I think foot is too generic, please edit and remove." and "rfoot on the other hand is fine." So I have removed it foot from the file. If there are other comments, I did not receive any. You can leave a comment here with a list of what needs to be fixed.

**Comment by andrewkm** [comment]
Weird I guess what I was trying to do on github didn't work and you didnt see my comments lol.

Anyways what I wrote in other places was:

"You cannot add duplicates into the file. There is already an sdoor for 330,0"

and

"All of these ones at the end of the file need to be organized to fit with our current ordering."

File: items.csv [hunk 3]
```diff
@@ -7204,13 +7212,19 @@ orangebanner,425,14
 whitebanner,425,15
 sprucedoor,193,0
 sprucedooritem,427,0
+sdoor,427,0
```
Comment (SirChilledMrHobo) [inline items.csv]: My mistake, sdoor 330 is an Iron Door. I'll change sdoor to "SPdoor".

File: items.csv [hunk 4]
```diff
@@ -7885,3 +7899,16 @@ cd12,2267,0
...
+bboat,445,0
+jungleboat,446,0
+jboat,446,0
+acaciaboat,447,0
+aboat,447,0
+darkoakboat,448,0
+doboat,448,0
+darkboat,448,0
+oakboat,333,0
+oboat,333,0
```
Comment (SirChilledMrHobo) [inline items.csv]: Sure thing.

**Comment by SirChilledMrHobo** [comment]
No worries. For now, this just have comments here to keep it simple. 👍

**Comment by andrewkm** [comment]
Merged, thank you very much :)

**Pull request merged**